packages/hydrogen/src/pagination/Pagination.ts

```diff
@@ -134,6 +141,12 @@ export function Pagination<NodesType>({
   });
 }
 
+function getNodes<NodesType>(connection: Connection<NodesType>) {
```

Comment:
You just... reimplemented `flattenConnection()`? It does this internally for you, doesn't it?